client/gutenberg/extensions/simple-payments/product-placeholder.jsx

```diff
@@ -0,0 +1,56 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './view.scss';
+
+export default ( { title = '', content = '', formattedPrice = '', multiple = false } ) => {
+	const assetUrl =
+		typeof window !== undefined && window.Jetpack_Block_Assets_Base_Url
+			? window.Jetpack_Block_Assets_Base_Url
+			: '/wp-content/plugins/jetpack/_inc/blocks';
+
+	return (
+		<div className="jetpack-simple-payments-wrapper">
+			<div className="jetpack-simple-payments-product">
+				<div className="jetpack-simple-payments-details">
+					{ title ? (
```

Comment:
I don't think we are against using `{ foo $$ <Bar /> }` anymore (and let Prettier format it)

```suggestion
{ title && <div className="jetpack-simple-payments-title"><p>{ title }</p></div> }
```